modules/core/src/main/scala/lucuma/odb/api/schema/TargetMutation.scala

```diff
@@ -141,7 +141,8 @@ trait TargetMutation extends TargetScalars {
   val InputObjectTypeCreateSidereal: InputObjectType[TargetModel.CreateSidereal] =
     deriveInputObjectType[TargetModel.CreateSidereal](
       InputObjectTypeName("CreateSiderealInput"),
-      InputObjectTypeDescription("Sidereal target parameters")
+      InputObjectTypeDescription("Sidereal target parameters"),
+      DocumentInputField("properVelocity", "Deprecated, use properMotion instead.")
```

Comment:
This is not very satisfactory, but there seems to be no way to mark an input field deprecated.